apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-steps.js

```diff
@@ -72,6 +72,7 @@ const referenceElements = [
 function getTourSteps( localeSlug, referencePositioning ) {
 	return [
 		{
+			id: 'welcome',
```

Comment:
I think this will fail the type check if used directly for the steps - since we changed the approach and no longer expect `id` on the Step. So we'd either need to pre-process before use or cast/assert as Step. 

I think ok to keep if absolutely needed for something else, but it might be worth reconsidering too.

Reply:
Thanks @chriskmnds, that was totally something that needed to be changed. I pushed a new commit that should solve it.